src/filefinder.cpp

```diff
@@ -462,36 +462,52 @@ void FileFinder::InitRtpPaths(bool warn_no_rtp_found) {
 
 	add_rtp_path(cs + "/" + version_str + "/");
 #else
+	// Windows/Wine
+	std::string const product = "RPG" + version_str;
 	if (Player::IsRPG2k()) {
 		// Prefer original 2000 RTP over Kadokawa, because there is no
 		// reliable way to detect this engine and much more 2k games
 		// use the non-English version
-		read_rtp_registry("ASCII", version_str, "RuntimePackagePath");
-		read_rtp_registry("KADOKAWA", version_str, "RuntimePackagePath");
+		read_rtp_registry("ASCII", product, "RuntimePackagePath");
+		read_rtp_registry("KADOKAWA", product, "RuntimePackagePath");
 	}
 	else if (Player::IsRPG2k3Legacy()) {
 		// Original 2003 RTP installer registry key is upper case
 		// and Wine registry is case insensitive but new 2k3v1.10 installer is not
 		// Prefer Enterbrain RTP over Kadokawa for old RPG2k3 (search order)
-		read_rtp_registry("Enterbrain", version_str, "RUNTIMEPACKAGEPATH");
-		read_rtp_registry("KADOKAWA", version_str, "RuntimePackagePath");
+		read_rtp_registry("Enterbrain", product, "RUNTIMEPACKAGEPATH");
+		read_rtp_registry("KADOKAWA", product, "RuntimePackagePath");
 	}
 	else if (Player::IsRPG2k3E()) {
 		// Prefer Kadokawa RTP over Enterbrain for new RPG2k3
-		read_rtp_registry("KADOKAWA", version_str, "RuntimePackagePath");
-		read_rtp_registry("Enterbrain", version_str, "RUNTIMEPACKAGEPATH");
+		read_rtp_registry("KADOKAWA", product, "RuntimePackagePath");
+		read_rtp_registry("Enterbrain", product, "RUNTIMEPACKAGEPATH");
 	}
 
+	// Our RTP is for all engines
+	read_rtp_registry("EasyRPG", "RTP", "path");
+
 	add_rtp_path("/data/rtp/" + version_str + "/");
 #endif
+	std::vector<std::string> env_paths;
+
+	// Windows paths are split by semicolon, Unix paths by colon
+	std::function<bool(char32_t)> f = [](char32_t t) {
+		return t == ';' || t == ':';
```

Comment:
This must be if-defed because Splitting at ":" will damage the path on Windows due to the drive letters being C:\